analyzer/src/funTest/kotlin/BabelTest.kt

```diff
@@ -51,7 +51,7 @@ class BabelTest : WordSpec() {
                 val nodeModulesDir = File(it, "node_modules")
                 val gitKeepFile = File(nodeModulesDir, ".gitkeep")
                 if (nodeModulesDir.isDirectory && !gitKeepFile.isFile) {
-                    nodeModulesDir.safeDeleteRecursively()
+                    nodeModulesDir.safeDeleteRecursively(true)
```

Comment:
Could use named argument `force = true` for better readability, boolean arguments can be confusing. Or make another helper `forceDeleteRecursively() = safeDeleteRecursively(true)` to make it shorter?

Reply:
I went for the named argument as I find `forceDeleteRecursively()` confusing because it is still *safe* WRT not following symbolic links / junctions.